### PR TITLE
Fix vote issues

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

**Comment by Raemon** [comment]
Seems slightly better if you factor out the sortings in review2019 and voting2019.

**Comment by Discordius** [comment]
Done

File: packages/lesswrong/components/review/ReviewVotingPage.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import seedrandom from '../../lib/seedrandom';
 
 const YEAR = 2019
 const NOMINATIONS_VIEW = "nominations2019"
-const REVIEWS_VIEW = "reviews2019" // unfortunately this can't just inhereit from YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
+const REVIEWS_VIEW = "voting2019" // unfortunately this can't just inhereit from YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
```
Comment (Raemon) [inline packages/lesswrong/components/review/ReviewVotingPage.tsx:23]: hmm, should probably also change this to "VOTING_VIEW" or just "VIEW"

File: packages/lesswrong/lib/collections/posts/views.ts [hunk 2]
```diff
@@ -1087,26 +1087,38 @@ ensureIndex(Posts,
...
+      nominationCount2019: { $gte: 2 }
+    },
+    options: {
+      sort: { ...(terms.sortBy && reviewSortings[terms.sortBy]), nominationCount2019: -1 }
+    }
+  }
+})
+
+Posts.addView("voting2019", (terms: PostsViewTerms) => {
   const sortings: Record<ReviewSortings, MongoSort<DbPost>> = {
```
Comment (Raemon) [inline packages/lesswrong/lib/collections/posts/views.ts:1122]: didn't actually abstract out the sortings here, despite having done so for other views.

Comment (Raemon) [reply packages/lesswrong/lib/collections/posts/views.ts:1122]: (see also the bottom of the voting2019 view, where it still uses "sortings")

File: packages/lesswrong/lib/collections/posts/views.ts [hunk 3]
```diff
@@ -1073,7 +1073,7 @@ Posts.addView("nominations2019", (terms: PostsViewTerms) => {
   return {
     selector: {
       // FIXME: Should only apply during voting
```
Comment (Raemon) [inline packages/lesswrong/lib/collections/posts/views.ts:1075]: remove this comment

**Review by Raemon — APPROVED** [review]
Okay. I'd doublecheck it actually works right in all the relevant places, but approved.

**Pull request merged**